azure/terraform/README.md

```diff
@@ -31,10 +31,34 @@ This is used to help generate unique resource names for:
 * Azure storage account name
 Note: these resources must be globally unique across all of Azure.
 
+The template also provides the `deploy_airflow` and `deploy_argo` flags. These are booleans that specify if airflow or argo will be deployed in the kubernetes cluster along with metaflow related services. By default `deploy_argo` is set to __true__ and `deploy_airflow` is set to __false__.
+
 Next, apply the `infra` module (creates Azure cloud resources only).
 
     terraform apply -target="module.infra"
 
+## Orchestrating Metaflow jobs on Kubernetes
+
+The recommended way to orchestrate Metaflow workloads on Kubernetes is via [Argo Workflows](https://docs.metaflow.org/going-to-production-with-metaflow/scheduling-metaflow-flows/scheduling-with-argo-workflows). However, Airflow is also supported as an alternative.
+
+### Argo
+Argo Workflows is installed by default on the AKS cluster as part of the `services` submodule. Setting the `deploy_argo` [variable](./variables.tf) will deploy argo in the AKS cluster. Not additional configuration is done in the `infra` module to support `argo`.
+
+### Airflow 
+
+**This is not a production deployment template**
```

Comment:
Then what is it?  Perhaps something like:

"This is quickstart template only, not recommended for real production deployments"